setup.cfg

```diff
@@ -0,0 +1,10 @@
+[metadata]
+
+name = 'nesta_ds_utils'
+version ='0.1'
+description = 'Utility package for Nesta Data Science Practise',
+url = 'https://github.com/nestauk/nesta_ds_utils',
+author = 'Nesta DAP',
+author_email = enrico.gavagnin@nesta.org.uk,
```

Comment:
1. Is there a reason to use quote marks everywhere else but not here?
2. Is this best served by a personal email or the team email address?

Reply:
Those were both very good points. Thank you for that.